packages/spreadsheet-view/src/SpreadsheetView/components/Spreadsheet.js

```diff
@@ -1,3 +1,8 @@
+import ArrowUp from '@material-ui/icons/ArrowUpward'
+import ArrowDown from '@material-ui/icons/ArrowDownward'
```

Comment:
Should be keyboard arrows
```suggestion
import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp'
import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
```